Components: comply with WCAG criterias

saabi opened the pull request:
Closes #275

mindrones commented on packages/components/time_region_value/src/node_modules/components/Timeline.svelte:
```diff
@@ -79,7 +82,10 @@
 					{#if hrefNext}
-						<a href={hrefNext}>
+						<a
+							aria-label={nextYear}
```
I'd use:
```suggestion
							aria-label='Show the next year ({nextYear})'
```

mindrones commented on packages/components/time_region_value/src/node_modules/components/Timeline.svelte:
```diff
@@ -57,7 +57,10 @@
 					{#if hrefPrev}
-						<a href={hrefPrev}>
+						<a
+							aria-label={prevYear}
```
I'd use:
```suggestion
							aria-label='Show the previous year ({prevYear})'
```

mindrones reviewed (commented):
Left some comments below.
Re: timeline nav buttons, when the button is supposed not to be clickable we might have these blocks:

```
<rect
	height={halfHeight}
	width={halfHeight}
/>
<polyline points={chevronLeftPath} />
```

within a `g` and add an aria label specifying the button is disabled.

saabi committed:
Components: Comply with WCAG criterias

Closes #275

- 1.3.1 G141 - Organizing a page using headings https://www.w3.org/WAI/WCAG21/Techniques/general/G141
- 1.3.1 H71 - Providing a description for groups of form controls using fieldset and legend elements https://www.w3.org/WAI/WCAG21/Techniques/html/H71
- 1.4.6 G17 - Ensuring that a contrast ratio of at least 7:1 exists between text (and images of text) and background behind the text https://www.w3.org/WAI/WCAG21/Techniques/general/G17
- 4.1.2 H91 - Using HTML form controls and links: https://www.w3.org/WAI/WCAG21/Techniques/html/H91

mindrones reviewed (approved):
Good to merge
